### ISSUE TITLE
Lack of accessibility features

### ISSUE THREAD

**Issue by Meghanaa020299** [Meghanaa020299/assignment3#4]
Absence or inadequacy of features that ensure accessibility for individuals with disabilities, impeding their ability to use and benefit from the system.
